docs/1.getting-started/5.third-parties.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Third Parties
+description: Learn how to optimize the performance of third-party resources using built-in composables and components.
+navigation.icon: i-ph-users-duotone
+---
+
+Nuxt provides a number of composables and built-in libraries that make it easier, and faster, to load third-party resources in your application.
+
+- `useScript`: Load any third-party script with server-side rendering support and a proxied API.
+- `useStyle`: Load any third-party stylesheet using various asset strategies.
+- Third-Party Wrappers: Wrapper components and composables that make it easier to include different popular third parties efficiently.
+
+## How Third Parties Can Impact User Experience
+
+Third parties are external resources included, but not directly controlled, by a site owner to add new functionality to a website. Popular examples of third parties include analytics, video embeds, maps, and social media integrations. Typically, third-party providers offer code snippets that can be added to the `head` or `body` section of the document.
+
+Adding a single third-party resource to your Nuxt application might not have any noticeable impact on performance, but it can quickly begin to affect user experience if you’re not careful. Many third parties, especially scripts, can take a relatively long time to download and execute, which can delay user interactivity and block page rendering.
+
+Data from the Chrome User Experience Report shows that Nuxt sites that load more third-party resources have lower [Interaction to Next Paint](https://web.dev/articles/inp) (INP) and [Largest Contentful Paint](https://web.dev/articles/lcp) (LCP) pass rates.
+
+![Chrome User Experience Report chart](/assets/docs/getting-started/third-parties/chart.png)
+<sub>
+source: Chrome User Experience Report,
+date: October 2023,
+devices: phone
+</sub>
+
+While the chart's correlation does not automatically indicate causation, lab experiments and data from the [Web Almanac](https://almanac.httparchive.org/en/2022/third-parties) provide further evidence that third-party resources significantly affect page performance.
+
+## Optimizing Third Party Performance
+
+In general, it can be difficult to determine the most optimal way to load different third-party resources. Nuxt provides a number of utilities that include various approaches to improve both the developer and user experience of loading third parties.
```

Comment:
```suggestion
In general, it can be difficult to determine the most optimal way to load different third-party resources. Nuxt provides a number of utilities to improve both the developer and user experience of loading third parties.
```